crates/stark-felt/src/lib.rs

```diff
@@ -124,25 +130,27 @@ impl Felt {
         BitArray::new(limbs)
     }
 
-    /// Checks if `self` is equal to [Felt::Zero].
-    pub fn is_zero(&self) -> bool {
-        *self == Felt::ZERO
-    }
     /// Finite field division.
     pub fn field_div(&self, rhs: &NonZeroFelt) -> Self {
         Self(self.0 / rhs.0)
     }
 
-    /// Floor division.
+    /// Truncated quotient between `self` and `rhs`.
     pub fn floor_div(&self, rhs: &NonZeroFelt) -> Self {
         Self(FieldElement::from(
             &(self.0.representative().div_rem(&rhs.0.representative())).0,
         ))
     }
 
-    /// Multiplicative inverse.
+    /// Quotient and remainder between `self` and `rhs`.
+    pub fn div_rem(&self, rhs: &NonZeroFelt) -> (Self, Self) {
```

Comment:
Isn't this the https://docs.rs/num-traits/latest/num_traits/ops/euclid/trait.Euclid.html trait?

Reply:
Not exactly, since the trait only has methods to perform one or the other, which means if you need both values you'll have to perform the expensive big integer division twice (made worse by the fact we need to convert to and from representatives).

Reply:
I opened an issue: https://github.com/rust-num/num-traits/issues/290

If they agree I will come up with a pull request adding it to the repo.

But for now let's keep it as it is